### PR TITLE
79 add bar graph to show user result section wise on result details page

### PR DESCRIPTION
**Pull request by Dhruv-Copods**
Features Added:

- Bar Graph ( Time Vs Sections ) to Showcase Results in a Graphical manner

### CODE REVIEW

File: app/components/barGraph/barGraph.tsx [hunk 1]
```diff
@@ -1,45 +1,139 @@
...
+          text: 'Time (in min)',
+        },
+      },
+    ],
+    legend: {
+      shadow: false,
+    },
+    tooltip: {
+      shared: true,
+      // formatter: function () {
```
Comment (yash6102000) [inline app/components/barGraph/barGraph.tsx:75]: please remove commented code...if not going to use

File: app/components/barGraph/barGraph.tsx [hunk 2]
```diff
@@ -1,45 +1,139 @@
...
-  //     }
-  //   });
-  //   return Math.ceil(maxTime/60)
-  // }
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+import moment from 'moment'
+
+const BarGraph = ({ sectionWiseResult }: { sectionWiseResult: any }) => {
+  console.log(sectionWiseResult)
```
Comment (yash6102000) [inline app/components/barGraph/barGraph.tsx:6]: please remove the console

File: app/components/barGraph/barGraph.tsx [hunk 3]
```diff
@@ -1,45 +1,139 @@
...
+        getSectionsFromResult[j][k].section.id
+      ) {
+        result.push(Math.floor(getSectionsFromResult[j][k].timeInSeconds / 60))
+      }
+    }
+  }
+
+  // const getLabelData = (sectionName: string, resultKind: string) => {
+  //   const getRequiredSection = sectionWiseResult.find(
+  //     (result: any) => result.section.section.name === sectionName
```
Comment (yash6102000) [inline app/components/barGraph/barGraph.tsx:38]: please remove comments ..if not going to use

File: app/routes/results/groupByTests/$testId.attended.$candidateResultId.tsx [hunk 4]
```diff
@@ -18,6 +18,7 @@ export const loader: LoaderFunction = async ({ request, params }) => {
     testId: candidateResult?.testId as string,
     candidateTestId: candidateResult?.candidateTestId as string,
   })
+  console.log(sectionWiseResult)
```
Comment (yash6102000) [inline app/routes/results/groupByTests/$testId.attended.$candidateResultId.tsx:21]: please remove console

File: app/components/results/ResultDetails.tsx [hunk 5]
```diff
@@ -73,7 +74,7 @@ const ResultDetailsComponent = () => {
         })}
       </div>
       <Divider height="1px" />
-      <div className="flex gap-4">
+      <div className="flex gap-4 pb-5">
```
Comment (yash6102000) [inline app/components/results/ResultDetails.tsx:77]: gap should be 7 (28px)

File: app/components/assessment/SectionDetails.tsx [hunk 6]
```diff
@@ -37,7 +37,7 @@ const SectionDetails = () => {
         </div>
       </div>
       <Button
-        className="px-11"
+        className="w-36 px-6"
```
Comment (yash6102000) [inline app/components/assessment/SectionDetails.tsx:40]: in results-test-candidate-list-tab please remove height and use proper py and py (page-sectoinCardForResultDetail)

File: app/components/results/ResultDetails.tsx [hunk 7]
```diff
@@ -56,8 +57,8 @@ const ResultDetailsComponent = () => {
           </span>
         </div>
       </header>
-      <Divider height="1px" />
-      <BarGraph />
+      <Divider height="10px" />
+      <BarGraph sectionWiseResult={sectionWiseResult} />
       <Divider height="1px" />
```
Comment (yash6102000) [inline app/components/results/ResultDetails.tsx:62]: dividers not showing on smaller display like 1024px  / 1440 px

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**